src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2155,11 +2158,12 @@ def aks_get_credentials(cmd, client, resource_group_name, name, admin=False,
         raise CLIError("Fail to find kubeconfig file.")
 
 
+# To be consistent, values of ADDONS dict should all be lower case
 ADDONS = {
```

Comment:
can you add a simple unit test to ensure the values of this dictionary are lowercase?

Reply:
sure